lnencrypt/tor_test.go

```diff
@@ -0,0 +1,60 @@
+package lnencrypt
+
+import (
+	"testing"
+
+	"github.com/btcsuite/btcd/btcec"
+	"github.com/lightningnetwork/lnd/lntest/channels"
+	"github.com/lightningnetwork/lnd/lntest/mock"
+	"github.com/stretchr/testify/require"
+)
+
+// TestTorPrivateKey tests writing and reading a private key to disk.
+func TestTorPrivateKey(t *testing.T) {
+	t.Parallel()
+
+	testPrivKeyBytes := channels.AlicesPrivKey
+
+	testPrivKey, _ := btcec.PrivKeyFromBytes(btcec.S256(),
+		testPrivKeyBytes)
+
+	tmpPrivateKey := "thisisasecretkey"
+	tmpPrivateKeyPath := "/tmp/torprivatekey"
+	keyRing := &mock.SecretKeyRing{
+		RootKey: testPrivKey,
+	}
+
+	err := WriteTorPrivateKey(
+		tmpPrivateKeyPath,
+		tmpPrivateKey,
+		keyRing,
```

Comment:
Nit: our style is to use as much available line space as possible. If it all fits in one line great, otherwise we'd want
```
WriteTorPrivateKey(
    tmpPrivateKeyPath, tmpPrivateKey, keyRing,
)